tasks/bbr-trigger-existing-pipeline.yml

```diff
@@ -0,0 +1,36 @@
+platform: linux
+
+params:
+  BOSH_CA_CERT:
+  BOSH_CLIENT:
+  BOSH_CLIENT_SECRET:
+  BOSH_DEPLOYMENT:
+  BOSH_ENVIRONMENT:
+
+inputs:
+- name: bosh-cli
+
+run:
+  path: /bin/bash
+  args:
+  - -cex
+  - |
+    set -o pipefail
+
+    install -m 0755 bosh-cli/bosh-cli-*-linux-amd64 /usr/local/bin/bosh
+    apt update && apt install -y curl
+
+    web_url=http://$(bosh instances --dns | grep "web" | awk '{print $4}' | head -n1):8080
+
+    until curl --silent --fail "$web_url/api/v1/info"; do
+      echo "waiting for /api/v1/info..."
+      sleep 2
+    done
+
+    echo "installing fly..."
+    curl "$web_url/api/v1/cli?platform=linux&arch=amd64" -o /usr/local/bin/fly
+    chmod +x /usr/local/bin/fly
+
+    fly -t target login -c "$web_url" -u admin -p admin
+    fly -t target unpause-pipeline -p bbr-test-2
```

Comment:
Can we give the `bbr-test-2` pipeline a different name, one that doesn't make it seem temporary. Worried someone might delete it. We could also add it to the `reconfigure-pipeline` to help ensure it's persistence.

Reply:
Can we add the pipeline yaml for `bbr-test-2` in this repo too?